Source/Widgets/InfoBox/InfoBoxDescription.css

```diff
@@ -0,0 +1,42 @@
+body {
+    margin: 0;
+    padding: 0;
+}
+
+.cesium-infoBox-description {
+    font-family: sans-serif;
+    font-size: 13px;
+    padding: 4px 10px;
+    margin-right: 4px;
+    color: #edffff;
+}
+
+.cesium-infoBox-description table {
+    color: #edffff;
```

Comment:
@emackey I can see why you did this, but it breaks HTML in the KML branch because it forces existing table snippets inside of child divs to have the specified colored text.  I think any table styling should be opt-in (That's what we have `cesium-infoBox-defaultTable` for after all).  Any counter arguments to removing this?